third-party/dms-guild/classes/class-pale-master.xml

```diff
@@ -0,0 +1,1891 @@
+		</spellcasting>
+		<rules>
+			<grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS_SPELLCASTING_SLOTS_FULL" />
+			<grant type="Grants" id="ID_INTERNAL_RITUAL_CASTING" />
+			<grant type="Proficiency" id="ID_INTERNAL_PROFICIENCY_SPELLFOCUS_GROUP_ARCANE_FOCUS" />
+
+			<stat name="pale master:spellcasting:slots:1" value="2" level="1" />
+			<stat name="pale master:spellcasting:slots:1" value="1" level="2" />
+			<stat name="pale master:spellcasting:slots:1" value="1" level="3" />
+
+			<stat name="pale master:spellcasting:slots:2" value="2" level="3" />
+			<stat name="pale master:spellcasting:slots:2" value="1" level="4" />
+			<stat name="pale master:spellcasting:slots:3" value="2" level="5" />
+			<stat name="pale master:spellcasting:slots:3" value="1" level="6" />
+			<stat name="pale master:spellcasting:slots:4" value="1" level="7" />
+			<stat name="pale master:spellcasting:slots:4" value="1" level="8" />
+			<stat name="pale master:spellcasting:slots:4" value="1" level="9" />
+			<stat name="pale master:spellcasting:slots:5" value="1" level="9" />
+			<stat name="pale master:spellcasting:slots:5" value="1" level="10" />
+			<stat name="pale master:spellcasting:slots:5" value="1" level="18" />
+			<stat name="pale master:spellcasting:slots:6" value="1" level="11" />
+			<stat name="pale master:spellcasting:slots:6" value="1" level="19" />
+			<stat name="pale master:spellcasting:slots:7" value="1" level="13" />
+			<stat name="pale master:spellcasting:slots:7" value="1" level="20" />
+			<stat name="pale master:spellcasting:slots:8" value="1" level="15" />
+			<stat name="pale master:spellcasting:slots:9" value="1" level="17" />
+
+			<stat name="pale master:spellcasting:prepare" value="intelligence:modifier" />
+			<stat name="pale master:spellcasting:prepare" value="level:pale master" />
+
+			<select type="Spell" name="Cantrip (Pale Master)" supports="$(spellcasting:list), 0" level="1" number="3" spellcasting="Pale Master" />
+			<select type="Spell" name="Cantrip (Pale Master)" supports="$(spellcasting:list), 0" level="4" number="1" spellcasting="Pale Master" />
+			<select type="Spell" name="Cantrip (Pale Master)" supports="$(spellcasting:list), 0" level="10" number="1" spellcasting="Pale Master" />
+
+			<!-- Grant for all Necromancy Spells -->
+			<!-- Currently works only for Cantrips -->
+			<!-- <grant type="Class Feature" id="ID_DMSG_GPPM_CLASS_FEATURE_PALE_MASTER_SPELLCASTING_PALE_MASTER_NECROMANCY_EXTEND" /> -->
+		</rules>
+	</element>
+
+	<!-- Extend for all Necromancy Spells -->
+	<!-- Currently works only for Cantrips -->
+	<!-- <element name="Spellcasting: Necromancy Extend" type="Class Feature" source="The Pale Master" id="ID_DMSG_GPPM_CLASS_FEATURE_PALE_MASTER_SPELLCASTING_PALE_MASTER_NECROMANCY_EXTEND">
+		<compendium display="false" />
+		<sheet display="false" />
+		<spellcasting name="Pale Master" extend="true">
+			<extend>Necromancy</extend>
+		</spellcasting>
+	</element> -->

```

Comment:
it's something that i though i would leave there, so people could un-comment it in case when they want to get spells from other sources other than Grimlore Grimoire and PHB, sadly it works only for Cantrips. Should i un-comment it and leave as it is? Or just remove it?

Reply:
yeah, removing it is fine